classes/Cart.php

```diff
@@ -756,7 +756,7 @@ public function getProducts($refresh = false, $id_product = false, $id_country =
             $givenAwayProductsIds = [];
 
             // Do not recalculate in case of refresh
```

Comment:
The comment seems not sync with the `if`

Reply:
yea thanks for noticing. In fact i need an advice here. This if stmt was not making any sense, but it was made on purpose by someone, so idk if i didn't break anything :( . There is only one public action which changes the $splitFiftProductsQuantity to true (cart::getProductsWithSeparatedGifts), but it also forces the $refresh to be true, so the gifts were never recalulated..
So I've decided that was just a mistake and removed the "!" from refresh.
In this case I should also align the comment to absolute opposite (as in the code too.) "Recalculate in case of refresh" Do these changes make sense to you?

Reply:
For me, yes :+1: